src/site/_includes/js/eligibility.js

```diff
@@ -1589,6 +1589,14 @@ function complexImmigration(input,
     complexOptions.includes(input.immigrationStatus));
 }
 
+// Returns true if discharge status is other-than-honorable, bad conduct,
+// or dishonorable. Used to determine complex discharge flag.
+function complexDischarge(input,
+  complexOptions=['dishonorable', 'oth', 'bad-conduct']) {
```

Comment:
dishonorable should be left off this list.  This press release makes it more clear that the amendment applies to other-than-honorable and bad conduct discharges.

https://news.va.gov/press-room/va-rule-amending-regulations-discharge-determinations/